src/FluentHub.App/Views/MainPage.xaml.cs

```diff
@@ -131,7 +133,7 @@ private void OnTabViewSelectionChanged(object sender, TabViewSelectionChangedEve
 			RootFrameBorder.Content = e.NewSelectedItem?.Frame;
 		}
 
-		private void LeftSideNavigationViewOpenerButton_Click(object sender, RoutedEventArgs e)
+		public void LeftSideNavigationViewOpenerButton_Click(object sender, RoutedEventArgs e)
```

Comment:
Why

Reply:
Must be for the command to be accessed in the tab bar